Source/common/SNTConfigurator.m

```diff
@@ -654,6 +662,32 @@ - (BOOL)fcmEnabled {
   return (self.fcmProject.length && self.fcmEntity.length && self.fcmAPIKey.length);
 }
 
+
+///
+/// Returns YES if all of the necessary options are set to export metrics, NO
+/// otherwise.
+///
+- (BOOL) exportMetrics {
+  return self.configState[kMetricsFormat] != SNTMetricFormatTypeUnknown &&
+    ![self.configState[kMetricsURL] isEqualToString:@""];
+}
+
+- (SNTMetricFormatType)metricsFormat {
+  switch ([self.configState[kMetricsFormat] longLongValue]) {
```

Comment:
I'd keep the value that users set as a string (you're right, it's much clearer) and have this bit of code translating from that string to the enum value.

Reply:
I've changed it back to a string and normalized the values entered by the operator.